microsoft-365/admin/email/create-a-shared-mailbox.md

```diff
@@ -76,6 +76,30 @@ Before creating a shared mailbox, be sure to read [About shared mailboxes](about
 
 You have a shared mailbox and it includes a shared calendar. Now go on to the next step: block sign-in for the shared mailbox account.
 
+## Which permissions should you use?
+
+You can use the following permissions with a shared mailbox.
+
+- **Full Access**: The Full Access permission lets a user open the shared mailbox and act as the owner of that mailbox. After accessing the shared mailbox, a user can create calendar items; read, view, delete, and change email messages; create tasks and calendar contacts. However, a user with Full Access permission can't send email from the shared mailbox unless they also have Send As or Send on Behalf permission.
+
+- **Send As**: The Send As permission lets a user impersonate the shared mailbox when sending mail. For example, if Kweku logs into the shared mailbox Marketing Department and sends an email, it will look like the Marketing Department sent the email.
```

Comment:
```suggestion
- **Send As**: The Send As permission lets a user impersonate the shared mailbox when sending mail. For example, if Kweku logs into the Marketing Department shared mailbox and sends an email, it will look like the Marketing Department sent the email.
```